usage/rest-api/src/main/java/org/apache/brooklyn/rest/domain/CatalogItemSummary.java

```diff
@@ -121,6 +132,20 @@ public String getIconUrl() {
         return iconUrl;
     }
 
+    public Collection<Object> getTags() {
+        List<Object> result = new ArrayList<Object>();
+        for (Object t : tags) {
+            // TODO if we had access to a mapper we could use it to give better json
```

Comment:
Can you expand on your comment? It's not really the job of a summary class to configure mappers but technically it does have access to one. Should you be adding another class to `org.apache.brooklyn.rest.transform`?